testing/scripts/test_alibi_explain.py

```diff
@@ -47,6 +53,8 @@ def test_alibi_explain_anchor_tabular(self, namespace):
 
         retry_run(f"kubectl apply -f {spec} -n {namespace}")
 
+        wait_for_status(sdep_name, namespace)
+        wait_for_rollout(sdep_name, namespace, expected_deployments=2)
         wait_for_deployment(name, namespace)
```

Comment:
nit: `wait_for_deployment(name, namespace)` equivalents are called inside `wait_for_rollout` so second call is redundant.